helm/conjur-config-cluster-prep/templates/clusterrole.yaml

```diff
@@ -1,3 +1,4 @@
+# This ClusterRole defines the Kubernetes API access permissions that the Conjur authenticator will require in order to validate application identities.
```

Comment:
This comment should be moved below the next line (i.e. it should be inside the `{{- if .Values.authnK8s.clusterRole.create }}` conditional). Having this comment outside of the conditional is causing the CI tests to break for any test cases where the conditional is false. When the conditional is false, Helm will still render the comment but nothing else... and when it goes to install this standalone comment, it errors out because there's no API version provided in this comment-only manifest.

Reply:
Ohh makes sense!